### DISCUSSION TITLE
Documentation of Symbol search - Wrong example

### DISCUSSION THREAD

**Discussion by spier** [community/community#8968] [Other Feature Feedback, Questions, & Ideas] [answered]
Excited to try out the new code search! :)

Found a little bug that I hope you will find useful:

The examples used in the [documentation](https://cs.github.com/about/syntax#symbol) of the **Symbol qualifier** are wrong.
Likely a copy and paste error from "Organization qualifier".

<img width="1001" alt="Screenshot 2021-12-16 at 19 41 58 " src="https://user-images.githubusercontent.com/163029/146429954-6562fde4-1fb9-4ca6-94f8-eba418eebda1.png">

**Comment by look** [comment, marked as answer]
Thanks for the report! This is now fixed.
